docs/book/src/reference/ports.md

```diff
@@ -0,0 +1,7 @@
+## Default Ports used by Cluster API / Cluster API Provider
+
+The default ports used by the Cluster API and Cluster API Providers are:
+
+- For `metrics` port `8080`
+- For `webhook` port `9443`
+- For `health` port `9440`
```

Comment:
Let's translate this into a table

`| Name | Port Number | Description |`

The description section should have enough details on how to use the port, and add appropriate links to external resources, if any.

Let's also add a note at the end that tells users to double check providers documentation, because ports might be allocated differently.

Reply:
done